app.js

```diff
@@ -0,0 +1,17 @@
+const express = require('express');
+const mongoose = require('mongoose');
+const bodyParser = require('body-parser');
+const commentRoutes = require('./routes/comments');
+
+const app = express();
+
+mongoose.connect('mongodb://localhost:27017/mydb', { useNewUrlParser: true, useUnifiedTopology: true });
```

Comment:
As talked in the class the  useNewUrlParser: true, useUnifiedTopology: true }) is not needed.
Also should use .env I called the url `DB_CONNECT` as we did in class but I'm sure there is much better names anyway